### PR TITLE
Create new release 02.06.00

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #2571 

Many change appear with app refactor so we need to be explicit in changelog about breaking change and deprecated parameter.

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
its necessary to add information about docker to now what is necessary to do to migratate from 2.5.x to 2.6.x (which new parameter is necessary to activate collaborative document)

**Comment by PhilippeAccorsi** [comment]
Need review on changelog.

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
 
+# 2.6.0 /2020-01-xx
+
+### New Feature
+- Multi-upload file directly available in tracim frontend
+- Page title show where you are in Tracim
+- Its now possible to delete user and anonymise information about him/her with tracimcli command
+- Preview generator now support raw file and 3D file (working with xvfb in docker container)
+- With proxy apache: you can now used browser cache policy. If you used Tracim with Docker you need to backup and delete existing apache configuration file in `~/tracim/etc/` (default path with Tracim Docker image). New file is created on next startup with new parameter inside.
+- All app is now enabled/disabled directly with one parameter and its more simple to develop new app
```
Comment (inkhey) [inline changelog.md]:
```suggestion
- All apps are now enabled/disabled directly with one parameter and its more simple to develop new app
```

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
 
 
+# 2.6.0 /2020-01-xx
+
+### New Feature
+- Multi-upload file directly available in tracim frontend
+- Page title show where you are in Tracim
+- Its now possible to delete user and anonymise information about him/her with tracimcli command
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- It's now possible to delete user and anonymise their informations with tracimcli command
```

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+- Preview generator now support raw file and 3D file (working with xvfb in docker container)
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- Preview generator now supports raw file and 3D file (working with xvfb in docker container)
```

File: changelog.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+- With proxy apache: you can now used browser cache policy. If you used Tracim with Docker you need to backup and delete existing apache configuration file in `~/tracim/etc/` (default path with Tracim Docker image). New file is created on next startup with new parameter inside.
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- With proxy apache: you can now use browser cache policy. If you used Tracim with Docker you need to backup and delete existing apache configuration file in `~/tracim/etc/` (default path with Tracim Docker image). On next startup, a new file will be created with the new parameters.
```

File: changelog.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+- All apps are now enabled/disabled directly with one parameter and its more simple to develop new app
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- All apps are now enabled/disabled directly with one parameter and it's now easier to develop new app
```

File: changelog.md [hunk 6]
```diff
@@ -1,5 +1,32 @@
...
+
+### Fixed issues
+- Gallery: #2540, #2541, #2551, #2574, #2583 
+- Frontend: #1396, #1560, #1656, #2607, #2608, #2611, #2641
+- Backend: #2588, #2570, #2610, #2129, #1484, #2010
+
+### Breaking/Important change
+- `agenda`, `share_content`, `upload_permission` and `collaborative_document_edition` are now real applications, you can add/remove them from `app.enabled` list and its add/remove the associated feature
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- `agenda`, `share_content`, `upload_permission` and `collaborative_document_edition` are now Tracim's applications (stand alone and optional), you can add/remove them from `app.enabled` list
```

Comment (Skylsmoi) [reply changelog.md]: the "(stand alone and optional)" part need to be confirmed by backend dev.
The point is to tell some main features of what is a Tracim's app

File: changelog.md [hunk 7]
```diff
@@ -1,5 +1,32 @@
...
+- All apps are now enabled/disabled directly with one parameter and its more simple to develop new app
+
+### Fixed issues
+- Gallery: #2540, #2541, #2551, #2574, #2583 
+- Frontend: #1396, #1560, #1656, #2607, #2608, #2611, #2641
+- Backend: #2588, #2570, #2610, #2129, #1484, #2010
+
+### Breaking/Important change
+- `agenda`, `share_content`, `upload_permission` and `collaborative_document_edition` are now real applications, you can add/remove them from `app.enabled` list and its add/remove the associated feature 
+  - `agenda` app is enabled by default, so you must have `app.enabled` list according to your old `caldav.enabled`(deprecated parameter). If your old config use `caldav.enabled = False` you should remove `agenda` on `app.enabled` list and uncomment this list
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
  - `caldav.enabled` is now deprecated and agenda is enabled by default. To disable it, uncomment `app.enabled` parameter and put each enabled apps but agenda.
```

File: changelog.md [hunk 8]
```diff
@@ -1,5 +1,32 @@
...
+  - `upload_permission` and  `share_content` are enabled by default but they now MUST be in `app.enabled` list to work properly. Before 2.6.0 theses apps can't be disabled.
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
  - `upload_permission` and  `share_content` are enabled by default. Before 2.6.0 theses apps couldn't be disabled. To disable them, uncomment `app.enabled` parameter and put each enabled apps but `upload_permission` and/or  `share_content`.
```

File: changelog.md [hunk 9]
```diff
@@ -1,5 +1,32 @@
...
+  - `collaborative_document_edition` is now a real app, disabled by default. To enable it you should not anymore set `collaborative_document_edition.activated` (deprecated parameter), but add `collaborative_document_edition` in `app.enabled` list and uncomment parameter.
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
  - `collaborative_document_edition.activated` is now deprecated and is disabled by default. To enable it, uncomment `app.enabled` parameter and put each enabled apps, including `collaborative_document_edition`
```

File: changelog.md [hunk 10]
```diff
@@ -1,5 +1,32 @@
...
+- If you use `collaborative_document_edition` with Tracim on Docker, you need to add `-e collaborative_document_edition=1` when you start docker with `docker run` command or add `collaborative_document_edition=1` in your docker-compose.yml file
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- If you use `collaborative_document_edition` with Tracim on Docker, you need to add `-e collaborative_document_edition=1` when you start docker with `docker run` command. You can also add `collaborative_document_edition=1` in your docker-compose.yml file
```

File: changelog.md [hunk 11]
```diff
@@ -1,5 +1,32 @@
...
+
+### Other Changes
+- 2 deprecated ldap parameters: `ldap_base_url` and `ldap_base_dn` (not used by tracim backend code in oldest release)
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- 2 deprecated ldap parameters: `ldap_base_url` and `ldap_base_dn` (they where not implemented in Tracim's backend code so using them had no impact)
```

File: changelog.md [hunk 12]
```diff
@@ -1,5 +1,32 @@
...
+- 2 deprecated app parameters: `caldav.enabled` and `collaborative_document_edition.activated` (no more used with app refactor #617)
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- 2 deprecated app parameters: `caldav.enabled` and `collaborative_document_edition.activated`
```

**Pull request merged**